otherlibs/stdune/src/temp.ml

```diff
@@ -44,7 +44,9 @@ let create_temp_dir ?perms path =
   | Created -> Ok ()
   | Already_exists -> Error `Retry
   | Missing_parent_directory ->
-    Code_error.raise "[Temp.create_temp_dir] called in a non-existing directory" []
+    Code_error.raise
+      "[Temp.create_temp_dir] called in a non-existing directory"
```

Comment:
While we're at it, perhaps we should also improve the grammar. "Non-existing" should be "nonexistent".